{{cookiecutter.app_name}}/README.rst

```diff
@@ -135,3 +135,56 @@ should cache all your assets forever by including the following line
 in your ``settings.py``::
 
     SEND_FILE_MAX_AGE_DEFAULT = 31556926  # one year
+
+
+{%- if cookiecutter.deployment_on_heroku == "yes" %}
+Deployment on Heroku
+--------------------
+
+Before using automatic deployment on Heroku you have to add migrations to your repository.
+You can do it by using following commands ::
+
+    flask db init
```

Comment:
The quickstart guide already prompts the user to initialize Flask migrations; the new information here is instructing the user to commit the migrations repo that is created.

Two thoughts here: 

1. By default, I don't believe the created project is initialized as a git repo. Users should be prompted to `git init` before trying to run any other commands. (In the future, we could include a post-create hook to initialize the project as a git repo, but out of scope for this change)
1. The instructions for adding migrations are applicable to all users and are best to be included in a section of the readme that isn't conditionally rendered.

Reply:
I've added info in _Migrations_ section and referenced in _Heroku_ section. I've added info about that git is something you should know but I think we have to assume that user knows the basic ideas of git. What do you think about it?